Node.java

```diff
@@ -0,0 +1,19 @@
+package consistent;
+
+/**
+ * Created by deepakrtp on 02/04/17.
+ */
+public class Node {
+    int serverID;
```

Comment:
Always use the access modifier for variables inside the class instead of leaving default value to kick in.